pinot-core/src/main/java/org/apache/pinot/core/plan/maker/InstancePlanMakerImplV2.java

```diff
@@ -164,7 +164,8 @@ static boolean isFitForDictionaryBasedPlan(QueryContext queryContext, IndexSegme
     for (ExpressionContext expression : selectExpressions) {
       FunctionContext function = expression.getFunction();
       String functionName = function.getFunctionName();
-      if (!functionName.equals("min") && !functionName.equals("max") && !functionName.equals("minmaxrange")) {
+      if (!functionName.equals("min") && !functionName.equals("max") && !functionName.equals("minmaxrange")
```

Comment:
Would be good to use `AggregationFunctionType` instead of string name. Also, would be good to factor out as `isFunctionFitForDictionaryBasedComputation` in say `AggregationFunctionUtils`, for readability.

Reply:
Done